src/components/Card/Card.css

```diff
@@ -0,0 +1,9 @@
+.card {
+    height: 200px;
```

Comment:
this can potentially be limiting if there's a lot of text. better to use something like `min-width: 200px`